src/index.ts

```diff
@@ -1,5 +1,5 @@
 import {getExpressMiddleware} from "./plugins/express-plugin";
-import {lowerCaseObjectKeys} from "./utils";
+import {formatRequestHeaders} from "./utils";
 import {UP9HttpConnector} from "./http-connector";
 
 const POLL_INTERVAL_MS = 5000;
```

Comment:
I think the name here is weird, maybe it is UP9 sender or something but it does not monitor,
not super critical but worth to think about it